test/integration/postOrder.spec.ts

```diff
@@ -34,7 +34,7 @@ suite("SDK: order posting", () => {
 
     const sellOrder = {
       accountAddress: walletAddress,
-      startAmount: LISTING_AMOUNT,
+      startAmount: +LISTING_AMOUNT,
```

Comment:
maybe keep one as a string or it might be smart to test all variations of bignumberish inputs to ensure their resulting values are equal